client/src/components/Landing.js

```diff
@@ -33,7 +40,14 @@ function Landing() {
 					}
 				</ul>
 			</nav>
-
+			{
```

Comment:
As far I know, this `Alert` will pop up everytime I reload the page because my `user` will not be `null` as well as `success` will also be `true`. Change this. Bring a success state from react-router-dom using the state parameter in navigate function from the login page.